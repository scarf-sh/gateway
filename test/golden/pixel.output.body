<this could be your png>