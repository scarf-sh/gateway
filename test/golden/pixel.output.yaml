captures: '[ Just (PixelCapture "12345") ]'
headers:
  Cache-control: no-cache, no-store, must-revalidate
  Content-Length: '24'
  Content-Type: image/png
query: ?x-pxid=12345
status: 200
